src/content/recipes/next.md

```diff
@@ -77,60 +63,556 @@ export default {
   ],
 };
 ```
+## Framework Options
```

Comment:
I appreciate wanting to make this information more easily found than the current README. But I don't think it's a good fit for a recipe, which should be quick hitting and to the point.

We'll add this info in proper framework docs, which should be coming soon.

Reply:
Sounds great! I'll just remove it for now then